openmls/src/treesync/node/leaf_node/capabilities.rs

```diff
@@ -174,6 +179,56 @@ impl Capabilities {
     }
 }
 
+/// A helper for building [`Capabilities`]
+#[derive(Debug, Clone)]
+pub struct CapabilitiesBuilder(Capabilities);
```

Comment:
Didn't you want to add an integration or doc test or something on this to show how to use it? Not too important though. It seems pretty clear how to use it.

Reply:
Huh, I think I mixed that up with the external join config builder from the other PR, for that I added something to the book. But yes, I agree that this is pretty obvious.